app/src/pages/playground/ModelSupportedParamsFetcher.tsx

```diff
@@ -29,8 +33,9 @@ export const ModelSupportedParamsFetcher = ({
   const updateModelSupportedInvocationParameters = usePlaygroundContext(
     (state) => state.updateModelSupportedInvocationParameters
   );
-  const { modelInvocationParameters } =
-    useLazyLoadQuery<ModelSupportedParamsFetcherQuery>(
+  useEffect(() => {
+    fetchQuery<ModelSupportedParamsFetcherQuery>(
```

Comment:
this was not the cause but this was re-mounting every render and firing a network call. This at least only runs when the useEffect fires.